Samples/ResizableAndHidableSample/EasyNotifyPropertyChanged.cs

```diff
@@ -1,14 +1,12 @@
 ﻿using System.ComponentModel;
 using System.Runtime.CompilerServices;
-using JetBrains.Annotations;
 
 namespace NonFullScreenSample
 {
     public class EasyNotifyPropertyChanged : INotifyPropertyChanged
     {
         public event PropertyChangedEventHandler? PropertyChanged;
 
-        [NotifyPropertyChangedInvocator]
```

Comment:
Why?

Reply:
iirc this caused an error which didnt let me compile so i just removed it